### PR TITLE
[C++] Hashmap Implementation

### PR DESCRIPTION
**Pull request by ujjwal404**

### CODE REVIEW

**Comment by amanmehara** [comment]
Please have a look at https://github.com/amanmehara/programming/blob/master/CONTRIBUTING.md and make changes accordingly.
I'll approve and merge afterwards! 😃

**Comment by ujjwal404** [comment]
thanks for helping. I will make the required changes.

**Comment by ujjwal404** [comment]
Hi @amanmehara . I've made the required changes. Please have a look and let me know if it still requires any further changes. 😄

**Review by amanmehara — CHANGES_REQUESTED** [review]

**Review by ujjwal404 — COMMENTED** [review]
done

File: C++/Hashmaps/hashmap.h [hunk 1]
```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2018 Aman Mehara
```
[the remark this answers is not in the document]
Comment (ujjwal404) [reply C++/Hashmaps/hashmap.h]: Thanks for helping. I'll make sure such mistakes won't happen again in future.😄

**Pull request merged**